app/src/main/java/org/fossasia/openevent/app/utils/CurrencyUtils.java

```diff
@@ -1,24 +1,30 @@
 package org.fossasia.openevent.app.utils;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Currency;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
 import java.util.SortedMap;
 import java.util.TreeMap;
+import java.util.TreeSet;
+
+import javax.inject.Inject;
 
 import io.reactivex.Single;
 import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
 
 /**
  * Source: https://stackoverflow.com/a/15343675/3309666
  */
 public final class CurrencyUtils {
 
     private static SortedMap<Currency, Locale> currencyLocaleMap;
+    private static Map<String, String> countryCurrencyMap;
 
-    private CurrencyUtils() {
+    @Inject
+    public CurrencyUtils() {
```

Comment:
This is a static utils class, if it is getting injected, make sure each static instance and method is made non static